x-pack/test/security_solution_api_integration/test_suites/siem_migrations/utils/rules.ts

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import SuperTest from 'supertest';
+import type { Client } from '@elastic/elasticsearch';
+import { X_ELASTIC_INTERNAL_ORIGIN_REQUEST } from '@kbn/core-http-common';
+import { replaceParams } from '@kbn/openapi-common/shared';
+
+import { RuleMigration } from '@kbn/security-solution-plugin/common/siem_migrations/model/rule_migration.gen';
+import { INDEX_PATTERN as SIEM_MIGRATIONS_INDEX_PATTERN } from '@kbn/security-solution-plugin/server/lib/siem_migrations/rules/data/rule_migrations_data_service';
+import { SIEM_RULE_MIGRATION_PATH } from '@kbn/security-solution-plugin/common/siem_migrations/constants';
+import { GetRuleMigrationResponse } from '@kbn/security-solution-plugin/common/siem_migrations/model/api/rules/rule_migration.gen';
+
+const SIEM_MIGRATIONS_RULES_INDEX_PATTERN = `${SIEM_MIGRATIONS_INDEX_PATTERN}-rules-default`;
+
+export type RuleMigrationSavedObject = Omit<RuleMigration, 'id'>;
+
+const migrationRuleSavedObject: RuleMigrationSavedObject = {
```

Comment:
```suggestion
export type RuleMigrationDocument = Omit<RuleMigration, 'id'>;

const migrationRuleDocument: RuleMigrationDocument = {
```

Reply:
done in https://github.com/elastic/kibana/pull/206822/commits/3b23a40c87495b83e2d94225524b3b15dc3f6b92